core/domain/exp_services.py

```diff
@@ -1426,6 +1421,12 @@ def revert_exploration(
     # not add the committer of the revert to the list of contributors.
     update_exploration_summary(exploration_id, None)
 
+    if feconf.ENABLE_PLAYTHROUGHS:
+        current_exploration = get_exploration_by_id(
+            exploration_id, version=current_version)
+        stats_services.update_exp_issues_for_new_exp_version(
+            current_exploration, None, revert_to_version)
```

Comment:
Please use kwargs here and on line 900 to add documentation to these calls, something like: `exp_versions_diff=None, version_num=revert_to_version`

Reply:
Done.